### PR TITLE
annie: updated team members 2-18 and executives team

### PR DESCRIPTION
**Pull request by aanxniee**
### Implementation description

- Changed term to 1236 and updated team/role/picture link for existing members
- Created new object for new members

### Steps to test

1.

### What should reviewers focus on?

-

### Checklist

- [ ] My PR name is descriptive and in imperative tense
- [ ] I have run the linter
- [ ] I have requested a review from other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

**Comment by aanxniee** [comment]
made a new branch, old one had fucked cicd from conflicts and stuff idfk

**Review by Jess2772 — APPROVED** [review]
looks good

File: constants/members.json [hunk 1]
```diff
@@ -3188,30 +3196,30 @@
...
       "name": "Cindy Li",
       "role": "vp internal",
-      "term": 1235,
+      "term": 1236,
       "teams": ["executive"],
-      "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Fs23headshots%2FVPI_Cindy_Li.jpg?alt=media&token=0ea951b4-629e-4615-a5b4-19d22cbc5280&_gl=1*wajfif*_ga*MTUwMDEyNjY3My4xNjc0MDgyODE2*_ga_CW55HF8NVT*MTY4NTkwMDY2Mi4zLjEuMTY4NTkwMTk1NS4wLjAuMA.."
+      "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Ff23headshosts%2FVPI_Cindy_Li.jpg?alt=media&token=35611a3e-ea9b-4bb1-a2a8-963d9ac954c4"
     },
     {
       "name": "Shirley Fang",
```
Comment (HeetShah) [inline constants/members.json:3204]: She doesn't seem to be on the list for executive list. Ik she is on the pics list, but let's leave her `term` to `1235` for now

File: constants/members.json [hunk 2]
```diff
@@ -3412,9 +3420,9 @@
...
       "name": "Oditha Amarasinghe",
       "role": "developer",
-      "term": 1235,
+      "term": 1236,
       "teams": ["casa"],
-      "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Fs23headshots%2FDev_Oditha_Amarasinghe.jpeg?alt=media&token=1dc5861c-2a97-45b2-bfaa-f79c26701446&_gl=1*k8k38z*_ga*MTUwMDEyNjY3My4xNjc0MDgyODE2*_ga_CW55HF8NVT*MTY4NTkwMDY2Mi4zLjEuMTY4NTkwMjU5MS4wLjAuMA.."
+      "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Ff23headshosts%2FDev_Oditha_Amarasinghe.jpeg?alt=media&token=7e71b36a-3275-4f8e-b52c-6c77d73833b0"
     },
     {
       "name": "Chen Kai Yan",
```
Comment (HeetShah) [inline constants/members.json:3428]: I believe this term should be `1236`. They are still a developer for casa on the casa list

File: constants/members.json [hunk 3]
```diff
@@ -3429,6 +3437,111 @@
...
+      "role": "vp scoping",
+      "term": 1236,
+      "teams": ["executive"],
+      "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Fdefault.png?alt=media&token=fe95cc90-ba2b-4c04-a808-0f903cc8b519"
+    },
+    {
+      "name": "Mathily Kadam",
+      "role": "internal director",
+      "term": 1236,
+      "teams": ["internal director"],
```
Comment (HeetShah) [inline constants/members.json:3515]: Where is the `internal director` team? I think the role `internal director` should be under `teams: executive`. This applies for `Olivia Chan`, `Maggie Lu`, `Krithi Sodhi`, `Serena Poonawalla` as well. Thoughts?

**Review by HeetShah — CHANGES_REQUESTED** [review]
What do you think about the comments? Start a thread on the channel for any questions/comments/concerns you have and please link this PR👍

**Comment by aanxniee** [comment]
> What do you think about the comments? Start a thread on the channel for any questions/comments/concerns you have and please link this PR👍

**Review by HeetShah — APPROVED** [review]
Nice, looks good

**Pull request merged**